WORKSPACE

```diff
@@ -259,138 +259,126 @@ new_go_repository(
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_spf13_cobra",
     commit = "9c28e4bbd74e5c3ed7aacbc552b2cab7cfdfe744",
     importpath = "github.com/spf13/cobra",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_inconshreveable_mousetrap",
     commit = "76626ae9c91c4f2a10f34cad8ce83ea42c93bb75",
     importpath = "github.com/inconshreveable/mousetrap",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_cpuguy83_go_md2man",
     commit = "648eed146d3f3beacb64063cd0daae908015eebd",  # Mar 19, 2017 (no releases)
     importpath = "github.com/cpuguy83/go-md2man",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_russross_blackfriday",
     commit = "35eb537633d9950afc8ae7bdf0edb6134584e9fc",  # Mar 19, 2017 (no releases)
     importpath = "github.com/russross/blackfriday",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_shurcooL_sanitized_anchor_name",
     commit = "10ef21a441db47d8b13ebcc5fd2310f636973c77",  # Mar 19, 2017 (no releases)
     importpath = "github.com/shurcooL/sanitized_anchor_name",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_golang_sync",
     commit = "450f422ab23cf9881c94e2db30cac0eb1b7cf80c",
     importpath = "github.com/golang/sync",
 )
 
-new_go_repository(
+go_repository(
     name = "org_golang_google_api",
     commit = "48e49d1645e228d1c50c3d54fb476b2224477303",
     importpath = "google.golang.org/api",
 )
 
-new_go_repository(
+go_repository(
     name = "com_github_googleapis_gax_go",
     commit = "9af46dd5a1713e8b5cd71106287eba3cefdde50b",
     importpath = "github.com/googleapis/gax-go",
 )
 
-new_go_repository(
+go_repository(
     name = "org_golang_google_genproto",
     commit = "411e09b969b1170a9f0c467558eb4c4c110d9c77",
     importpath = "google.golang.org/genproto",
 )
 
 ##
-## Proxy build rules
+## Docker rules
 ##
 
-PROXY = "9a1bae7a5d947bb81a4898fbd171d129aeb04c52"
-
-http_file(
-    name = "istio_proxy",
-    url = "https://storage.googleapis.com/istio-build/proxy/envoy-alpha-" + PROXY + ".tar.gz",
+git_repository(
+    name = "io_bazel_rules_docker",
+    commit = "146c9b946159a8fafbf81723c40652f192ee56ac",  # Jul 21, 2017
+    remote = "https://github.com/bazelbuild/rules_docker.git",
 )
 
-http_file(
-    name = "istio_proxy_debug",
-    url = "https://storage.googleapis.com/istio-build/proxy/envoy-debug-" + PROXY + ".tar.gz",
+load(
+    "@io_bazel_rules_docker//docker:docker.bzl",
+    "docker_repositories",
+    "docker_pull",
 )
 
-##
-## Docker rules
-##
+docker_repositories()
 
-DEBUG_BASE_IMAGE_SHA = "3350805cd019192b1a71fcdcc786ac4a8500e0d4f96c30e5e4c8ff43b9208195"
+ISTIO_IPTABLES_TAG = "20170725-5a9fbbf"
 
-http_file(
-    name = "ubuntu_xenial_debug",
-    sha256 = DEBUG_BASE_IMAGE_SHA,
-    url = "https://storage.googleapis.com/istio-build/pilot/ubuntu_xenial_debug-" + DEBUG_BASE_IMAGE_SHA + ".tar.gz",
+docker_pull(
+    name = "istio_iptables",
```

Comment:
The init image is called gcr.io/istio-testing/init:_. I'm fine renaming that one to istio_init. This is the base that just contains iptables + xenial.